tutorials/time-freq/ssvep.py

```diff
@@ -241,29 +241,97 @@ def snr_spectrum(psd, noise_n_neighborfreqs=1, noise_skip_neighborfreqs=1):
 plt.xlim([0, fmax])
 fig.show()
 
+
+###############################################################################
+# SNR plotting function
+# ^^^^^^^^^^^^^^^^^^^^^^^^
+def plot_snr_spectrum(snrs, freqs, use_stem_plot=False, stim_freq=None, bg_var_trials=False, bg_var_channels=False,
```

Comment:
sorry for being a bit inconsistent with earlier, but i think *here* the function makes it more difficult than necessary :D 

the catches and further options are not necessary, if we dont use them in the tutorial, and the other plots are also written in plain code. 
..this would also make the `n_dimensions` fix obsolete (which is indeed the better name)

Reply:
No problem! I'll remove the function.